tanner/emulators/sqli.py

```diff
@@ -46,8 +48,10 @@ def map_query(self, attack_value):
                 error_result = 'You have an error in your SQL syntax; check the manual\
                                 that corresponds to your MySQL server version for the\
                                 right syntax to use near {} at line 1'.format(attack_value['id'])
+                self.logger.exception('Error while executing MySQL %s', error_result)
```

Comment:
why it's an exception? it's emulated error, not the tanner one :) maybe make it debug? also `error_result` can be logged without code duplication

Reply:
Got it!